docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -26,6 +27,512 @@
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `gene_symbol`
+		* New permissible value: `MPO`
+		* New permissible value: `RAS, NOS`
+	* Changes made to `hpv_strain`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `laboratory_test`
+		* New permissible value: `Abnormal Cells`
+	* Changes made to `molecular_analysis_method`
+		* New permissible value: `Immunophenotyping, NOS`
+	* Changes made to `molecular_consequence`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `mutation_codon`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `pathogenicity`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `second_gene_symbol`
+		* New permissible value: `MPO`
+		* New permissible value: `RAS, NOS`
+	* Changes made to `staining_intensity_scale`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `staining_intensity_value`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `test_units`
+		* New permissible value: `IU`
+		* New permissible value: `nmol/L`
+		* New permissible value: `Seconds`
+		* New permissible value: `x10^6 cells/mcL`
+	* Changes made to `test_value_range`
+		* New permissible value: `0-25%`
+		* New permissible value: `26-50%`
+		* New permissible value: `51-75%`
+		* New permissible value: `76-100%`
+		* New permissible value: `Unknown`
+		* New deprecated value: `0-25`
+		* New deprecated value: `26-50`
+		* New deprecated value: `51-75`
+		* New deprecated value: `76-100`
+	* Changes made to `timepoint_category`
+		* New permissible value: `Initial Diagnosis`
+		* New permissible value: `Sample Procurement`
+		* New permissible value: `Unknown`
+	* Changes made to `variant_origin`
+		* New permissible value: `Not Reported`
+* Altered `other_clinical_attribute` Entity
+	* New property: `days_to_risk_factor`
+	* New property: `hormonal_replacement_therapy_status`
+	* New property: `number_of_pregnancies`
+	* New property: `viral_hepatitis_serology_tests`
+	* Changes made to `eye_color`
+		* New permissible value: `Unknown`
+	* Changes made to `immunosuppressive_treatment_type`
+		* New permissible value: `Immunoglobulin`
+		* New permissible value: `Prednisone`
+	* Changes made to `nononcologic_therapeutic_agents`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `oxygen_use_type`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `pregnancy_outcome`
+		* New permissible value: `Full Term Birth, NOS`
+	* Changes made to `risk_factor_method_of_diagnosis`
+		* New permissible value: `Unknown`
+	* Changes made to `risk_factors`
+		* Changes made to `items`
```

Comment:
I recommend removing the "items" lines,  I saw a few throughout the notes.

Reply:
Done 👍